### ISSUE TITLE
Radium crashes on 1.18.1 server when using Ars Noveau's Summon Decoy Glyph

### ISSUE THREAD

**Issue by Kuusotare** [Asek3/Radium#14]
https://pastebin.com/5BH0zkeg

**Comment by WFPhantom** [comment]
can confirm that happens on 1.18.2
https://pastebin.com/Sg5hp0j7

**Comment by NBZion** [comment]
For now, maybe you can disable the decoy glyph from ars nouveau from radium to fix this certain crash.

**Comment by WFPhantom** [comment]
um pretty sure this has been fixed so no need to do that

**Comment by shinji257** [comment]
Nah.  This still seems to be present.  Pack TNP 5 2.15.0 using Ars Nouveau 2.6.5 and it still has this bug.  I'm going to look at disabling the glyph.

**Comment by WFPhantom** [comment]
@shinji257 that is still an outdated version of ars. Ars 2.7.4 with the patch note "fixes radium crash" should fix it

**Comment by shinji257** [comment]
> @shinji257 that is still an outdated version of ars. Ars 2.7.4 with the patch note "fixes radium crash" should fix it

If you are going to point out that it is "still an outdated version" and that a newer version fixed it, can you at least look at when I posted the message in question?  My message was posted on July 16th.  The update came out on August 16th which is a month later.

With that said (and I just checked) TNP does have the fixed version included in it now.  Probably did soon after the Ars dev released a fixed version.
